fetch_data.py

```diff
@@ -273,11 +316,14 @@ def main():
 
     try:
         search_query = options["<query>"].split(',')
-        are_users = True if options["users"] else False
+        if options["<mode>"] not in ["users", "search", "likes"]:
+            raise Exception("Mode must be one of 'users', 'search', 'likes'.")
```

Comment:
PTAL. Would like to know whether I should drop this check entirely, raise a different exception, and/or change the logic for mapping mode strings to Mode enum values.

Reply:
I think this looks good! 
- I don't know of any better way to convert from command line arguments to enum.
- I guess the check could be dropped for simplicity/readability, since using an invalid Mode should result in an understandable error, but it's still fine.